yaksh/test_views.py

```diff
@@ -97,6 +98,31 @@ def test_view_profile_get_for_user_with_profile(self):
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'yaksh/view_profile.html')
 
+    def test_email_verification_for_user_post(self):
+        """
+        POST request to verify email
+        """
+        self.client.login(
+            username=self.user2.username,
+            password=self.user2_plaintext_pass
+        )
+        post_response = self.client.post(reverse('yaksh:new_activation'),
+                data={'email':self.user2.email}
+            )
+        subject = mail.outbox[0].subject.replace(" ", "_")
+        activation_key = mail.outbox[0].body.split("\n")[2].split("/")[-1]
+        get_response = self.client.get(reverse('yaksh:activate',
```

Comment:
Although this is fine, my worry is on a machine with a SMTP server, this test will actually send a mail to the email id of the test user. But if we use a different email backend provided by Django, (like dummy or file backends) for development, it would be better. Maybe have a look into this?